catalog/spatial/csw/spatial-csw-endpoint/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/endpoint/transformer/CswRecordMapperFilterVisitor.java

```diff
@@ -479,6 +479,7 @@ private double getDistanceInMeters(double distance, String units) {
     return new Distance(distance, linearUnit).getAs(LinearUnit.METER);
   }
 
+  @Override
   @SuppressWarnings("squid:S2177" /* Doesn't override parent method */)
```

Comment:
❗ @emmberk, reminder to look into this

Reply:
I removed the suppress line because it does, in fact, override the parent method, regardless of whether `@Override` is present or not.